From: Ralf Treinen <treinen@debian.org>
Last-Update: 2011-08-06
Description: Set the default postscript viewer in xhtml2ps to "see"
 Upstream default doesn't fit well with a Debian system. Using see and
 lpr are a better default since they match commands available for
 Debian. Bug #72034 has a bigger discussion regarding the subject.

Index: html2ps-1.0b7/contrib/xhtml2ps/xhtml2ps
===================================================================
--- html2ps-1.0b7.orig/contrib/xhtml2ps/xhtml2ps	2011-01-10 02:41:55.000000000 -0200
+++ html2ps-1.0b7/contrib/xhtml2ps/xhtml2ps	2011-01-10 02:44:40.000000000 -0200
@@ -804,8 +804,8 @@
   array set stusr {
     command     "html2ps"
     checker     "weblint -x Netscape"
-    viewer      "ghostview -magstep -2 -a4"
-    printcmd    "lp"
+    viewer      "gv -media=a4 -scale -2"
+    printcmd    "lpr"
     filepcmd    "exec cp"
     orientation " "
     colonnes    " "
